railties/lib/rails/generators/rails/app/templates/Gemfile

```diff
@@ -32,7 +32,7 @@ end
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
-  <%- unless options.skip_system_test? || options.api? -%>
+  <%- unless options.skip_system_test? || options.skip_test? || options.api? -%>
```

Comment:
Can we create a method like `depend_on_listen` and move those conditionals there?